src/utils/CustomPropTypes.js

```diff
@@ -38,6 +54,10 @@ const CustomPropTypes = {
   all
 };
 
+function errMsg(props, propName, componentName) {
+  return `Invalid prop '${propName}' of value '${props[propName]}' supplied to '${componentName}'`;
```

Comment:
Ohh.. My bad :blush: 
I was going to add it.. but children.

Thank you for pointing it out ! :cherries: 
I'll fix it now.